nova_vm/src/ecmascript/abstract_operations/type_conversion.rs

```diff
@@ -829,15 +809,48 @@ pub(crate) fn to_big_int(
 }
 
 /// ### [7.1.14 StringToBigInt ( str )](https://tc39.es/ecma262/#sec-stringtobigint)
-pub(crate) fn string_to_big_int(_agent: &mut Agent, _argument: String) -> Option<BigInt> {
+pub(crate) fn string_to_big_int(
+    agent: &mut Agent,
+    gc: GcScope<'_, '_>,
```

Comment:
**suggestion:** Couldn't this be `NoGcScope<'_, '_>` because the only use of GC in this function is a `.nogc()` use? Then we also wouldn't need the `.reborrow()` in `nova_vm/src/ecmascript/abstract_operations/testing_and_comparison.rs:433`.

Reply:
Oh yeah, absolutely.

Reply:
Makes sense, I have addressed this. Thank you!